ScnSoft-Education/LeetCodeTraining/3SumClosest-16/3SumClosestSolution.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace LeetCodeTraining.ThreeSumClosest
+{
+    //https://leetcode.com/problems/3sum-closest/
+    public class ThreeSumClosestSolution
+    {
+        public static int ThreeSumClosest(int[] nums, int target)
+        {
+            Array.Sort(nums);
```

Comment:
Please add vertical alignment where it can lead to better code viewing:
grouping and separating initialization parameters from operations blocks with them, return statement in general also should be forwarded  by a blank line.